setup.py

```diff
@@ -8,7 +8,7 @@
 
 setup(
     name='django-auth-lti',
-    version='1.0',
+    version='2.0',
```

Comment:
Speaking of semver, it's not clear what aspect of the library's "public api" is changing that would warrant a major version increment anyway.